debvader/debvader.py

```diff
@@ -238,32 +229,19 @@ def iterative_deblending(net, field_image, galaxy_distances_to_center_in, npeaks
         shifts = np.concatenate((shifts_previous, shifts), axis = 0)
         cutout_images_total = np.concatenate((cutout_images_total, cutout_images), axis = 0)
         output_images_total = np.concatenate((output_images_total, output_images_mean), axis = 0)
-        galaxy_distances_to_center_total = np.concatenate((galaxy_distances_to_center_total, galaxy_distances_to_center), axis = 0)
+        nb_of_galaxies_in_deblended_field_total += nb_of_galaxies_in_deblended_field
         diff_mse = mse_step-mse_step_previous
         k+=1
-        if diff_mse==0.:
-            # If no galaxy is found here, except the ones that are too close from the borders, try to locate more galaxies.
-            npeaks_per_iteration+=10
-            detection_k = detect_objects(field_image, npeaks_per_iteration=npeaks_per_iteration, factor=factor)
-            # Avoid to have several detection at the same location
-            idx_to_remove = []
-            for i in range (len(detection_k)):
-                if detection_k[i] in galaxy_distances_to_center_total:
-                    idx_to_remove.append(i)
-            detection_k = np.delete(detection_k, idx_to_remove, axis = 0)
-            if npeaks_per_iteration==500:
-                print('converged on maximum peak per iteration.')
-                break
-
-        print(str(len(shifts))+' galaxies found up to this step.')
+
+        print(str(nb_of_galaxies_in_deblended_field_total)+' galaxies found up to this step.')
```

Comment:
```suggestion
        print(f'{nb_of_galaxies_in_deblended_field_total} galaxies found up to this step.')
```

Reply:
Ok I tried to use more f-strings. See [1387371](https://github.com/BastienArcelin/debvader/commit/138737139df0104610b68f98323b9dab51d393a1#diff-a5dae2bb532104d1ce10a8b52d5c651b023b7d0d57e87a12c0c6942e0d44cee7)